plugins/dra-ocp-validator/tools/install-nfd.sh

```
@@ -0,0 +1,233 @@
+#!/usr/bin/env bash
+# Install Node Feature Discovery (NFD) on OpenShift cluster
+# Required for GPU auto-detection
+
+set -euo pipefail
+
+# Retry helper for transient API errors
+retry_command() {
+    local max_attempts=3
+    local delay=5
+    local attempt=1
+    local cmd="$@"
```

Shellcheck (0.11.0)

[warning] Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

(SC2124)